src/datumaro/plugins/anchor_generator.py

```diff
@@ -183,7 +183,7 @@ def get_anchors(
                 x_center + 0.5 * ws,
                 y_center + 0.5 * hs,
             ]
-            base_anchors = torch.stack(base_anchors, dim=-1)
+            base_anchors = torch.stack(base_anchors, dim=-1).to(self.device)
```

Comment:
Is this related to the purpose of this PR?

Reply:
Nope, this is to fix anchor generator.. I will make a separate PR for fixing this. Thank you for pointing out this.